specification/loadtestservice/LoadTestService/examples/2022-11-01/CreateOrUpdateTest.json

```diff
@@ -0,0 +1,183 @@
+{
+  "operationId": "LoadTestAdministration_CreateOrUpdateTest",
```

Comment:
Q: why do they show as new? Didnt they exist already?

Reply:
Due to new TypeSpec generated spec being totally out of sync compared to old one it shows as new changes.